package.json

```diff
@@ -266,5 +272,16 @@
     "typescript": {
       "memoryLimit": 4096
     }
-  }
+  },
+  "prettier": {
+  "bracketSpacing": false,
+  "bracketSameLine": false,
+  "printWidth": 90,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "arrowParens": "avoid"
+}
```

Comment:
Ironic but did this part of `package.json` not get auto-formatted? Also, since prettier is no longer formating JS files (right?) is this necessary?

Reply:
It didn't get auto-formatted because the previous prettier config was `"include": ["static/app", "tests/js"],`. I think we should revisit it to handle all possible files. Nice catch.

This is still required since premier is used at pre-commit hooks formatting for markdown and yaml

Reply:
Apparently Biome considers `package.json` a [protected file](https://biomejs.dev/guides/how-biome-works/#protected-files) which is fair enough 🤔 

As for running Prettier for Markdown and Yaml, gotcha, makes sense, though it seems weird to have formatting settings that don't affect JavaScript in `package.json` rather than a `prettierrc` or whatever. Not a big deal